test/test_ops.py

```diff
@@ -192,6 +228,48 @@ def wrapped_fn(*args, **kwargs):
 
             self.assertEqual(result, expected)
 
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_jvp', set({
+        # Apprently these support forward AD, but we get "Trying to use forward AD..."
+        xfail('var_mean'),
+        xfail('std_mean'),
+
+        # RuntimeError: expected scalar type double but found float
+        xfail('minimum'),
+        xfail('min', 'binary'),
+        xfail('maximum'),
+        xfail('max', 'binary'),
```

Comment:
I'm getting `NotImplementedError: Cannot access storage of TensorWrapper` on my branch after patching in your stack from https://github.com/pytorch/pytorch/pull/67041 and this PR, am I doing something wrong?

Reply:
Nevermind, the problem is I am building in debug mode and there are those autograd asserts...